docs/domains.rst

```diff
@@ -141,10 +141,15 @@ The ``/api/v1/domains/`` endpoint reponds to ``GET`` requests with an array of
     curl -X GET https://desec.io/api/v1/domains/ \
         --header "Authorization: Token {token}"
 
-to retrieve an overview of the domains you own.
+to retrieve an overview of the domains you own.  Domains are returned in
+reverse chronological order of their creation.
```

Comment:
Does our code actually guarantee this? I am not aware of any sorting.

Reply:
This is a new side-effect feature: The paginator paginates based on the created timestamp.